### DISCUSSION TITLE
New Project Idea for GSoC 23

### DISCUSSION THREAD

**Discussion by rahul-ws** [cncf/mentoring#895] [Google Summer of Code 2023]
How can a contributors propose there own ideas for GSoC which are not included in idea list?

**Comment by nate-double-u** [comment]
Hi @rahul-ws, we don't recommend contributors propose their own ideas and don't have a process to support it. We ask maintainers to come up with project ideas so that we can ensure that any work being done can be supported by the community.

If you have an idea you think would make for a good project, I suggest reaching out to that project community and discussing it with them, and if they also think it's a good idea, and have the capacity to mentor that project, then they can open a PR to include the concept in the list and join the program.
